prz-ml/reserv
# add basic tests

- register USERS blueprint
- add pytest to requirements
- move source to src folder
- fix import path after moving
- add emoji to the header of readme
- remove misplaced app folder
- add workflow file
- add codecov badge
- add .vim/ to .gitignore
- added data mockup
- add env-based configurations
- create app factory
- check if there are any blueprints to register
- add information about optional arguments
- create basic fixture for client
- create proper testing configuration
